modules/test/playwright/pages/FDS/FDS.page.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/applicationsMenu.page';
+
+import type {Page} from '@playwright/test';
+
+export class FDSPage {
+	page: Page;
+	applicationsMenuPage: ApplicationsMenuPage;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+	}
+
+	async goto() {
+		await this.applicationsMenuPage.goToFDS();
+	}
+
+	async createTestDataSet() {
+		await this.goto();
+
+		await this.page
+			.getByRole('button', {
+				name: /New Data Set/,
```

Comment:
What are these slashes? :thinking: Are these regex expressions? That would probably be an overkill to put everywhere. Any reason why not simple single quotes?

Reply:
Looks like it is better to go with `string`. 
Probably use the locator through the Playwright UI or copy-paste another locator around